README.md

```diff
@@ -104,11 +100,13 @@ charge of mapping spoken language from an audio file into a series of phonemes,
 distinguish one word from another. In order to make this process as accurate as possible, the phoneme factory also
 receives the user text file and language selection.
 
-Once the phoneme file is created, it is sent off to a secondary process called the phoneme processor. This processor
-maps phonemes from the phoneme file to mouth assets received from a directory. There will be 1:1 mapping between
-phonemes and mouth assets. These assets will be added onto the avatar and rendered into a video file. The video file
-will be sent back to the front end and will be available for the user to download onto their local machine.
+Once the phoneme file is created, it is sent off to a secondary process called the phoneme processor. This processor maps phonemes from the phoneme file 
+to a specific timestamp, and associates that timestamp with an asset file containing the phoneme, an avatar pose, and state of blinking. There is a 1:1 
+mapping between phonemes and mouth assets. These assets are then compiled and rendered into a video file using ffmpeg. The video file is then returned to 
+the front end, and is available for the user to download onto their local machine.
 
 ## Limitations
 
-TBD
+- The phoneme generation process controlled via Rhubarb can be quite computationally intensive, and is the current bottleneck for performance.
+- Blink frequency is chosen using a random probability
```

Comment:
Is blinking based on probability a limitation or is it just something we can refine after peer evaluations?